R/derive_adeg_params.R

```diff
@@ -16,7 +16,7 @@
 #'   parameter) and to the parameters specified by `qt_code` and `rr_code`.
 #'
 #'
-#' @param by_vars Grouping variables
+#' @param by_vars `r roxygen_param_by_var(arg = 1)`
 #'
 #'   Only variables specified in `by_vars` will be populated
```

Comment:
@pharmaverse-bot main question primarily is do we want to consolidate/standardize the additional text for each `by_vars` entry that varies incredibly in detail for each individual function too?